src/nvim/sign.c

```diff
@@ -873,6 +881,22 @@ static void sign_list_append_info(sign_T *sp, MTKey *mark, list_T *retlist)
   }
 }
 
+/// Append dictionary of information for a defined sign "sp", or placed
+/// sign "mark" to "retlist". Either "sp", or "mark" is NULL.
```

Comment:
If this function is necessary the docstring here and at `sign_list_append_info()` should be updated.

Reply:
done, also simplified the arguments further.